modules/runners/scale-down.tf

```
@@ -18,6 +18,7 @@ resource "aws_lambda_function" "scale_down" {
       RUNNER_BOOT_TIME_IN_MINUTES             = var.runner_boot_time_in_minutes
       SCALE_DOWN_CONFIG                       = jsonencode(var.idle_config)
       GHES_URL                                = var.ghes_url
+      NODE_TLS_REJECT_UNAUTHORIZED            = var.ghes_url != null && !var.ghes_ssl_verify ? 0 : 1
```

Fix terraform format
```suggestion
      NODE_TLS_REJECT_UNAUTHORIZED            = var.ghes_url != null && ! var.ghes_ssl_verify ? 0 : 1
```